histolab/filters/morphological_filters.py

```diff
@@ -24,10 +24,8 @@
 from . import morphological_filters_functional as F
 from .image_filters import Filter
 
-try:
-    from typing import Protocol, runtime_checkable
-except ImportError:
-    from typing_extensions import Protocol, runtime_checkable
+
+from typing_extensions import Protocol, runtime_checkable
```

Comment:
I think that we were using this try/except because we are still supporting python 3.7, and Protocol has been introduced in Python 3.8

Reply:
yes, that's why I'm importing it directly from typing exceptions, it is a dependency of histolab os it is fine to just do that.

Doing the try catch will complicate the typing quite a bit unfortunately

Reply:
fair enough